pkg/ddevapp/config_test.go

```diff
@@ -564,7 +564,7 @@ func TestConfigValidate(t *testing.T) {
 	app.DdevVersionConstraint = ">= 1.23"
 	err = app.ValidateConfig()
 	assert.Error(err)
-	assert.Contains(err.Error(), "project has a DDEV version constraint of '>= 1.23' and the version of DDEV you are using ('v1.22.3-11-g8baef014e') does not meet the constraint")
+	assert.Contains(err.Error(), "your DDEV version 'v1.22.3-11-g8baef014e' doesn't meet the constraint '>= 1.23'")
```

Comment:
This set of comparisons would be easier to create and manage with a test array and t.Run(). I've recently started using that more for repetitive things in tests and like it quite a lot. For example, https://github.com/ddev/ddev/blob/c2aca52a18687e678086dd232573cf51914dba56/pkg/ddevapp/config_test.go#L1545-L1568

Reply:
That's a good point.

Reply:
Done.